packages/dx-react-scheduler/src/plugins/day-view.tsx

```diff
@@ -70,79 +71,79 @@ class DayViewBase extends React.PureComponent<VerticalViewProps, ViewState> {
     timeTableRowComponent: 'TimeTableRow',
   };
 
-  viewCellsDataBaseComputed = ({
-    currentDate,
-  }) => {
-    const { intervalCount, startDayHour, endDayHour, cellDuration } = this.props;
-    return viewCellsDataCore(
-      currentDate, undefined,
-      intervalCount, [],
-      startDayHour!, endDayHour!, cellDuration!,
-      Date.now(),
-    );
-  }
-
   timeTableElementComputed = () => this.timeTable;
   layoutElementComputed = () => this.layout;
   layoutHeaderElementComputed = () => this.layoutHeader;
 
-  layoutHeaderElement: ComputedFn = (getters) => {
-    const { name: viewName } = this.props;
+  layoutHeaderElement = viewName => (getters) => {
     return computed(
       getters, viewName!, this.layoutHeaderElementComputed, getters.layoutHeaderElement,
     );
   }
+  memoLayoutHeaderElement = memoize(this.layoutHeaderElement);
```

Comment:
It's better to rename memo -> memoized to avoid confusion with React.Memo

Reply:
I used the `memo` word to reduce length.